src/artifacts-credprovider-conda/src/activate/artifacts-cred-activate.bat

```diff
@@ -0,0 +1,4 @@
+echo Running artifacts-cred-provider-conda...
```

Comment:
Let's try to avoid writing text to stdout in these scripts. Also, for the .bat ones, add `@echo off` to the top so they don't print out the commands as they get executed

Reply:
Removed print statement from all scripts